easy-rules-core/src/main/java/org/jeasy/rules/core/DefaultRuleListener.java

```diff
@@ -56,11 +56,12 @@ public void beforeExecute(Rule rule, Facts facts) {
 
     @Override
     public void onSuccess(Rule rule, Facts facts) {
-        LOGGER.log(Level.INFO, "Rule ''{0}'' performed successfully", rule.getName());
+        LOGGER.info("Rule ''{}'' performed successfully", rule.getName());
     }
 
     @Override
     public void onFailure(Rule rule, Facts facts, Exception exception) {
-        LOGGER.log(Level.SEVERE, String.format("Rule '%s' performed with error", rule.getName()), exception);
+        LOGGER.info("Rule '{}' performed with error: {}",
+            rule.getName(), exception.getMessage());
```

Comment:
The exception should not be swallowed here. We should log the whole exception, not only the message. Do you agree?

Reply:
Totally agree ;)

Reply:
Note that slf4j doesn't have natively a method with string replacement and exception logging. 
For this case we need manually concat the message (in this case checking first if is the log enabled avoiding concatenation when the log is off)